other/sign-release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# set -e
+TAG=$(git describe --abbrev=0 --tags)
+USER=${USER:-$(git config --get user.name)}
+VERSION=${TAG/v/}
+
+echo "Going to verify and sign releases."
+echo ""
+echo ""
+
+git diff --exit-code >> /dev/null || (echo "Working Dir not clean, this script won't work." && false)
+
+echo ""
+echo "Getting $TAG.zip"
+curl -LOs "https://github.com/TokTok/c-toxcore/archive/$TAG.zip"
+unzip -q "$TAG.zip"
+cp -r "c-toxcore-$VERSION"/* .
```

Comment:
This does not cover a modifies online source were files are missing. The better solution is to compare both tar files via cmp:
https://github.com/NicoHood/gpgit/commit/1862f7971999ca774b9e03e719b84e0f0957f66a#diff-7a66bd62453af6c2a03d2b155a00662bR471